### PR TITLE
LPS-165113 - When using the App Pages, the Picklist field does not display it's default value

### PR DESCRIPTION
**Pull request by ruben-pulido**
//cc @ealonso

### CODE REVIEW

**Comment by ruben-pulido** [comment]
ci:test:sf

**Comment by ruben-pulido** [comment]
ci:test:relevant

File: modules/apps/info/info-api/src/main/java/com/liferay/info/field/type/SelectInfoFieldType.java [hunk 1]
```diff
@@ -40,9 +40,20 @@ public String getName() {
 
 	public static class Option {
 
+		public Option(
+			boolean isDefault, InfoLocalizedValue<String> label, String value) {
+
+			_label = label;
+			_value = value;
+
+			_default = isDefault;
```
Comment (ealonso) [inline modules/apps/info/info-api/src/main/java/com/liferay/info/field/type/SelectInfoFieldType.java:49]: @ruben-pulido it should be named default, we don't use isDefault for fields, it is used only for method names

Comment (ruben-pulido) [reply modules/apps/info/info-api/src/main/java/com/liferay/info/field/type/SelectInfoFieldType.java:49]: default is a reserved word in java, so we cannot use it.

I hesitated between calling it default_ or isDefault and decided to use isDefault

**Comment by ruben-pulido** [comment]
https://github.com/liferay-echo/liferay-portal/pull/10173

**Pull request closed without merging**